### PR TITLE
Add new headers for code.quarkus.io requests

### LINKED ISSUE #130: Add new header parameters when calling code.quarkus.io
```
API consumers shoult add two new header params:
Client-Name: the name of the client (ex: "Code Quarkus")
Client-Contact-Email: the email to contact in case of change in the API
```

Client-Name: vscode-quarkus
Client-Contact-Email: fbricon (use redhat email)

@ia3andy should we add the version in the client name? Can we send multiple contact headers?

### PR DESCRIPTION
**Pull request by xorye**
Fixes #130 

Signed-off-by: David Kwon <dakwon@redhat.com>

### CODE REVIEW

File: src/utils/requestUtils.ts [hunk 1]
```diff
@@ -78,8 +85,16 @@ export async function downloadProject(state: ProjectGenState): Promise<ZipFile>
 }
 
 async function tryGetProjectBuffer(projectUrl: string): Promise<Buffer> {
+  const requestOptions: request.OptionsWithUri = {
+    uri: projectUrl,
+    headers: {
+      'Client-Name': CLIENT_NAME,
+      'Client-Contact-Email': CLIENT_CONTACT_EMAIL
```
Comment (fbricon) [inline src/utils/requestUtils.ts]: is it possible to define a headers constant holding those values? to avoid repeating it for each query

Comment (xorye) [reply src/utils/requestUtils.ts]: I couldnt find anything in the docs: https://www.npmjs.com/package/request#custom-http-headers

Should I put the `request()` in a custom wrapper class to allow that?

Comment (fbricon) [reply src/utils/requestUtils.ts]: simply do 
```ts
const HEADERS = {
  'Client-Name': 'vscode-quarkus',
  'Client-Contact-Email': 'fbricon@redhat.com,azerr@redhat.com,dakwon@redhat.com'
};
...

const requestOptions: request.OptionsWithUri = {
    uri: `${apiUrl}/extensions`,
    headers: HEADERS,
    timeout: 30000
  };
...
```

Comment (xorye) [reply src/utils/requestUtils.ts]: thanks, I've pushed the change

**Pull request merged**